docs/user-guide/compute-resources.md

```diff
@@ -159,8 +164,8 @@ The [resource quota](../admin/resource-quota.md) feature can be configured
 to limit the total amount of resources that can be consumed.  If used in conjunction
 with namespaces, it can prevent one team from hogging all the resources.
 
-### Detecting Resource Starved Containers
-To check if a container is being killed because it is hitting a resource limit, call `kubectl describe pod`
+### My container is terminated 
```

Comment:
The users may not know that their container is having problem because it's resource-starved, so I think putting the symptom in the headline makes it easier for them to find the solution.